apps/src/sharedJsxStyles.js

```diff
@@ -6,10 +6,6 @@
 
 module.exports = {
   colors: {
-    charcoal: '#5b6770',
-    green: '#b9bf15',
-    white: '#fff',
-    orange: '#ffa400',
```

Comment:
Would it make more sense to get rid of this file completely, and add remaining colors to the scss file?